src/components/Gnb.jsx

```diff
@@ -1,20 +1,37 @@
+import { useState, useEffect } from 'react';
+
+import { getUser } from '../apis/api';
 import logoImg from '../assets/logo.svg';
 
 import './Gnb.css';
 
-const Gnb = ({items}) => {
-  const isShowProfile = items && items.profileImageSource && items.email;
+const Gnb = ({className}) => {
+  const [users, setUsers] = useState({});
+
+  useEffect(() => {
+    const fetchUser = async () => {
+      const { data } = await getUser();
+      
+      setUsers(data[0]); 
+      // 하드 코딩 이유: 아마, 회원 인증에서 처리해 users의 넘버를 넘겨 유저들 배열 중에서 특정 유저에 대한 정보로 접근할 듯합니다.
+      // 따라서, 현재로서는 동적으로 접근하는 방법이 떠오르지 않아, 하드 코딩을 진행했습니다.
+    };
```

Comment:
response값을 봤는데 지금 구조에서는 이렇게 해도 괜찮을 것 같네요!
그리고 배열의 인덱스에 직접 접근하는 것도 지양하는 편이라서 아래처럼 표현해줘도 괜찮을 것 같아요~
```suggestion
      const { data } = await getUser();
      const [user] = data;
      
      setUsers(user); 
      // 하드 코딩 이유: 아마, 회원 인증에서 처리해 users의 넘버를 넘겨 유저들 배열 중에서 특정 유저에 대한 정보로 접근할 듯합니다.
      // 따라서, 현재로서는 동적으로 접근하는 방법이 떠오르지 않아, 하드 코딩을 진행했습니다.
    };
```